### PR TITLE
ibm-label <label> element does not have descriptive text that identif…

### PR DESCRIPTION
**Pull request by maheshkanchetti**
…ies the expected input

Closes IBM/carbon-components-angular#

{{short description}}

#### Changelog

**New**

* {{new thing}}

**Changed**

* {{change thing}}

**Removed**

* {{removed thing}}

### CODE REVIEW

File: src/input/label.component.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ import { TextArea } from "./text-area.directive";
 	template: `
 		<label
 			[for]="labelInputID"
+			[attr.aria-label]="labelInputID"
```
Comment (cal-smith) [inline src/input/label.component.ts:41]: Generally we want to add a `ariaLabel` `@Input` for this 👍 The `labelInputID` is just to identify this label, it's not meant to be human readable.

**Review by cal-smith — CHANGES_REQUESTED** [review]
Thanks for taking this on! Looks good overall, but one small suggestion!

File: src/input/label.component.ts (hunk 1 above)
Comment (maheshkanchetti) [reply src/input/label.component.ts]: Thanks, Updated it has Input.

**Review by cal-smith — APPROVED** [review]
Thanks! LGTM 💯

**Review by theiliad — APPROVED** [review]

**Comment by maheshkanchetti** [comment]
@cal-smith can you please merge the changes.

**Comment by cal-smith** [comment]
Yep! Just going through and merging PRs now 🙂

**Pull request merged**